bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandler.java

```diff
@@ -174,4 +175,24 @@
      */
     void handleRemoval();
 
+    /**
+     * This method should return a unique identifier of the device handled by this {@link ThingHandler}.
```

Comment:
> introducing getRepresentationProperty() would not help 

Why not? `handler.getThing().getProperties().get(hander.getRepresentationProperty())` would always return an individual result, wouldn't it?
But looking at that statement, I wonder if the RepresentationProperty shouldn't rather be a feature of the Thing and not of the Handler. Wdyt?